lib/commands/source.js

```diff
@@ -37,6 +42,70 @@ helpers.mobileGetSource = async function (opts = {}) {
   return await this.proxyCommand(`/source?format=${encodeURIComponent(opts.format)}`, 'GET');
 };
 
+/* Will get JSON of the form:
+ *   { isEnabled: '1',
+ *     isVisible: '1',
+ *     frame: '{{0, 0}, {375, 667}}',
+ *     children:
+ *      [ { isEnabled: '1',
+ *          isVisible: '1',
+ *          frame: '{{0, 0}, {375, 667}}',
+ *          children: [],
+ *          rect: { x: 0, y: 0, width: 375, height: 667 },
+ *          value: null,
+ *          label: null,
+ *          type: 'Other',
+ *          name: null,
+ *          rawIdentifier: null },
+ *     rect: { origin: { x: 0, y: 0 }, size: { width: 375, height: 667 } },
+ *     value: null,
+ *     label: 'UICatalog',
+ *     type: 'Application',
+ *     name: 'UICatalog',
+ *     rawIdentifier: null }
+ */
+function getTreeForXML (srcTree) {
+  function getTree (element, elementIndex, parentPath) {
+    let curPath = `${parentPath}/${elementIndex}`;
+    let rect = element.rect || {};
+    let subtree = {
+      '@': {
+        name: element.name || '',
+        label: element.label || '',
+        value: element.value,
+        dom: null,
+        enabled: parseInt(element.isEnabled, 10) === 1,
```

Comment:
in the original xml it's either `true` or `false`

Reply:
This comes out in the XML result as something like
```xml
<XCUIElementTypeOther name="No SIM" label="No SIM" value="" dom="" enabled="true" valid="true" visible="true" hint="" path="/0/2/0/1/0" x="7" y="0" width="54" height="27">
```